src/colmap/scene/database.cc

```diff
@@ -1339,9 +1367,10 @@ void Database::CreateImageTable() const {
 void Database::CreatePosePriorTable() const {
   const std::string sql =
       "CREATE TABLE IF NOT EXISTS pose_priors"
-      "   (image_id          INTEGER  PRIMARY KEY  NOT NULL,"
-      "    position          BLOB,"
-      "    coordinate_system INTEGER               NOT NULL,"
+      "   (image_id                   INTEGER  PRIMARY KEY  NOT NULL,"
+      "    position                   BLOB,"
+      "    position_covariance        BLOB,"
```

Comment:
Would you mind updating the Python database interface to add the new column? https://github.com/colmap/colmap/blob/9c1ad104d2d54dc18996d1ac7a0d4befb9e91b9b/scripts/python/database.py#L211

Reply:
Done :+1: 
I have also added some tests in the example_usage() function of the scripts.

Reply:
Nice, thank you!